app/services/symphony_client.rb

```diff
@@ -93,6 +93,44 @@ def cancel_hold(holdkey, session_token)
                           })
   end
 
+  def place_hold(patron, session_token, item_barcode, hold_args)
+    body = {
+      "itemBarcode": item_barcode,
```

Comment:
surprised that rubocop didn't complain about double quotes here 🤔

Reply:
oh yeah 🤔  idk why not, I will fix that.

Reply:
don't feel compelled, I'm just genuinely surprised